.travis.yml

```diff
@@ -24,6 +24,8 @@ before_script:
   # Install cx_Oracle (which depends on Oracle IC)
   - cd ./../backend/uclapi
   - pip3 install $(cat requirements.txt | grep "cx-Oracle")
+  - sudo apt-get -y install redis-server
+  - sudo service redis-server restart
```

Comment:
Do we need these two lines? We already have redis set up as a `service` on line 7
Docs for that: https://docs.travis-ci.com/user/database-setup/#Redis